manifests/proxy.pp

```diff
@@ -429,7 +429,7 @@
   if ($listenip != undef) {
     if ($listenip =~ /^(eth|lo|bond|lxc|eno|tap|tun).*/) {
       $listen_ip = getvar("::ipaddress_${listenip}")
-    } elsif is_ip_address($listenip) {
+    } elsif $listenip =~ Stdlib::Compat::Ip_address {
```

Comment:
Should this use `Stdlib::IP::Address` instead of `Stdlib::Compat::Ip_address`?

Reply:
`metadata.json` requires `stdlib >= 4.19.0 < 6.0.0`.
Data type `Stdlib::IP::Address` requires  at least `stdlib 4.25.0`.

Reply:
@Dan33l That's the reason why I chose `Stlib::Compat::Ip_address` :) If its ok to increase the stdlib dependency to 4.25.0, I'm happy to use `Stdlib::IP::Address`.

Reply:
I'd be fine with increasing the stdlib dependency.